templates/index.html

```diff
@@ -233,6 +233,7 @@ <h3><u>Contact Us</u></h3>
           submit_button.disabled = false;
           submit_button.innerHTML = "Submit";
           submit_button.style.opacity = 1;
+          submit_button.style.padding = 10;
```

Comment:
Why is this added in? Not seeing any difference between when I leave it in and take it out